apps/portals/src/portal-components/challengeportal/ChallengeDetailPageWrapper.tsx

```diff
@@ -11,7 +15,23 @@ import { AppUtils, ChallengeDetailPage } from 'synapse-react-client'
  */
 const ChallengeDetailPageWrapper = () => {
   const projectId = AppUtils.useQuerySearchParam('id')
-  if (projectId) {
+  const { data, isLoading } = SynapseQueries.useGetEntityBundle(
```

Comment:
https://sagebionetworks.jira.com/browse/PORTALS-2935

Reply:
Since this is conditionally shown based on an annotation value, I thought the portal wrapper would be the appropriate place for this check.